### ISSUE TITLE
Test proposal for base

### ISSUE THREAD

**Issue by GantMan** [infinitered/ignite#83]
- [x] Issue does not already exist
- Identify exactly what aspect your issue is about
  - [x] ignite base applicattion
  - [ ] ignite the cli
  - [ ] ignite generated code

We should add fixtures for our example API, along with a test that verifies the functions available on APIFixtures at least match the functions designated on API live.   This helps stop a situation where someone renames or utilizes one and not the other. Result: a large time delay between the dev who worked on a project and maybe even had their PR accepted, and the network error they would cause.

This test is valuable in generated projects.

I'd like to see @fvonhoven implement this.   Thoughts welcome @skellock

**Comment by GantMan** [comment]
FYI,  API is hard to bring into mocha tests :(   So much stubbing needed.

**Comment by kevinvangelder** [comment]
@GantMan: @skellock and I came up with a reasonable way to handle this. Basically, we have parallel services, Api which calls the real api and FakeApi which calls the fixtures, and then in our saga index we override any erroring or absent endpoints with the fake method. For testing, we simply load up FakeApi and use it directly. It probably sounds clunky how I've described it, but in action it's actually pretty slick.
